go.mod

```diff
@@ -68,4 +68,6 @@ replace git.schwanenlied.me/yawning/bsaes.git => github.com/Yawning/bsaes v0.0.0
 // btcsuite/btcutil package requests a newer version.
 replace golang.org/x/crypto => golang.org/x/crypto v0.0.0-20190211182817-74369b46fc67
 
+replace github.com/lightningnetwork/lightning-onion => github.com/guggero/lightning-onion v1.0.2-0.20200428130542-d019eaf53e32
```

Comment:
Can be updated to just point to onion master. We should also start to tag versions for the lib.